### PR TITLE
Update for tests

### PR DESCRIPTION
**Pull request by nrobakova**
### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [x] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { IgxToggleModule } from '../directives/toggle/toggle.directive';
 import { IBaseCancelableBrowserEventArgs } from '../core/utils';
 
 /* eslint-disable @typescript-eslint/no-use-before-define */
-describe('IgxTimePicker', () => {
+xdescribe('IgxTimePicker', () => {
```
Comment (kdinev) [inline projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts:18]: I think this was leftover by accident?

**Review by kdinev — CHANGES_REQUESTED** [review]

File: projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts (hunk 1 above)
Comment (nrobakova) [reply projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts:18]: no :) this tests are failing with my changes, that's why I exclude them :) Nikolay said that they refactor the IgxTimePicker tests and will try with refactored tests when they are pushed

Comment (kdinev) [reply projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts:18]: Hmmm, OK!

Comment (Lipata) [reply projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts:18]: @kdinev we prefer to fix those tests in the pickers refactoring branch, so when this PR is merged to master, then we will update the pirkcers refactoring branch and will remove the xdescribe

Comment (nrobakova) [reply projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts:18]: @kdinev if we include this failing tests the grid test won't be executed that's why I just exclude them

Comment (Lipata) [reply projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts]: New time-picker tests are merged and enabled and if there are some problems with the build we will fix them

**Review by Lipata — APPROVED** [review]

**Review by kdinev — APPROVED** [review]

**Pull request merged**